justfile

```diff
@@ -161,7 +161,7 @@ rust-build: _codegen-order-workaround
 build: rust-build dart-build
 
 # Tests dart (builds all necessary parts)
-dart-test: rust-build dart-build
+dart-test: rust-build dart-build download_assets
```

Comment:
We don't want to download the assets every time we run the tests.

Reply:
With the curl flag `-C -` we don't download the assets again if they have already been downlaoded. 
> Use "-C -" to tell curl to automatically find out where/how to resume the transfer. It then uses the given
              output/input files to figure that out.

Is that ok? otherwise we should at least put it in the readme

Reply:
This also avoid the unpack of the archive?

Reply:
another point, should we add a command to delete them in `clean-fully`?

Reply:
unfortunately no. We can add the flag `--keep-old-files` but it doesn't make it noticeably faster. verifying the checksum takes the longest. we can also just add an if and check if the archive exist if not then we fetch, extract and verify. wdyt?

Reply:
> another point, should we add a command to delete them in clean-fully?

done